common/app/services/newsletters/NewsletterApi.scala

```diff
@@ -10,21 +10,24 @@ import scala.concurrent.duration.DurationInt
 import scala.concurrent.{ExecutionContext, Future}
 
 case class NewsletterResponse(
-    id: String,
+    identityName: String,
     name: String,
     brazeNewsletterName: String,
     brazeSubscribeAttributeName: String,
     brazeSubscribeEventNamePrefix: String,
     theme: String,
     description: String,
     frequency: String,
-    exactTargetListId: Int,
-    listIdv1: Int,
+    listIdV1: Int,
     listId: Int,
     exampleUrl: Option[String],
     emailEmbed: EmailEmbed,
     illustration: Option[NewsletterIllustration] = None,
     signupPage: Option[String],
+    restricted: Boolean,
+    paused: Boolean,
+    emailConfirmation: Boolean,
```

Comment:
what does this boolean indicate? Sorry I know it comes from the spread sheet but I can't quite guess what it's for